Buscorepuestos/buscorepuestos-web
# Feature/br 353 card valoration

## What type of PR is this? (check all applicable)


## Description
The component was redesigned and its responsive mobile version was developed. It is worth mentioning that breakpoints had to be placed in other swiper components so that none of them overflowed the other or disrupted their styles.


## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-353/card-valoration

## QA Instructions, Screenshots, Recordings

Run the app with pnpm run dev and only see port: 3000, check the card valoration component and its responsive

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
